src/Middleware/RequestThrottling/src/RequestThrottlingMiddleware.cs

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.RequestThrottling;
+using Microsoft.AspNetCore.RequestThrottling.Internal;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Aspnetcore.RequestThrottling
+{
+    /// <summary>
+    /// Limits the flow of requests through your application,
```

Comment:
"Flow" is another somewhat vague term. Better to be clear about precisely what is limited:

```suggestion
    /// Limits the number of concurrent requests allowed in the application,
```